python/cudf/tests/test_joining.py

```diff
@@ -398,3 +398,25 @@ def test_dataframe_pairs_of_triples(pairs, max, rows, how):
         for column in gdf_result:
             assert np.array_equal(gdf_result[column].fillna(-1).sort_values(),
                                   pdf_result[column].fillna(-1).sort_values())
+
+
+def test_crash():
```

Comment:
Can this be renamed to give it a more descriptive name? Something about doing a left join when the left table is empty?

Reply:
Of course! I didn't know you'd want to use mine for the final fix. :)

Reply:
Yeah, I really enjoy the process of when a bug is found, that the reporter can open a PR with a unit test that reproduces the bug, and then I can just commit the fix to that same PR.